database/entity/0069-add_user_roles_table/UserRole.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, ManyToOne, JoinColumn } from 'typeorm'
+import { User } from '../User'
+
+@Entity('user_roles', { engine: 'InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci' })
+export class UserRole extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({ name: 'user_id', type: 'int', unsigned: true, nullable: false })
+  userId: number
+
+  @Column({ length: 40, nullable: false, collation: 'utf8mb4_unicode_ci' })
+  role: string
+
+  @Column({ name: 'created_at', default: () => 'CURRENT_TIMESTAMP', nullable: false })
```

Comment:
Durch den default muss der Wert nicht in der UserRole gefüllt werden, diese wird dann beim Save automatisch gesetzt

Reply:
yes that's right. in the migration the default is set correctly, but was forgotten here in the entity. Thank you for that hint.